integration_tests/int_inbound_tests.py

```diff
@@ -28,19 +28,29 @@ def setUp(self):
             max_retries=int(config.get_config('INBOUND_QUEUE_MAX_RETRIES', default='3')),
             retry_delay=int(config.get_config('INBOUND_QUEUE_RETRY_DELAY', default='100')) / 1000)
 
-    def get_message_body(self, message):
-        json_message = json.loads(message.body)
-        message_payload = json_message['payload']
-        split_payload = message_payload.split(',')
-        message_body = split_payload[3]
-        return message_body
-
     @test_utilities.async_test
     async def test_acceptance_transaction(self):
-        mymessage = 'this is my message'
-        await self.incoming_queue.send_async(mymessage)
+        edifact_file = """UNB+UNOA:2+GP123+HA456+200427:1737+00000045'
```

Comment:
can this not be stored in a file ?

Reply:
we test here parsing edifact to fhir, we do not need to exclude it from file. The message will be received as string. So it would be better to rename this variable into edifact_string or edifact_message than storying it in a file. @mattd-kainos , @Broom94 how do you think?

Reply:
MD says leave it inline